sdk/src/extensions.rs

```diff
@@ -76,6 +76,7 @@ impl FieldTypeExt for FieldType {
                 quote!(#prefix #model::Data)
             }
             Self::Scalar(typ, _) => typ.to_tokens(),
+            Self::Unsupported(_) => unreachable!("Unsupported fields are not generated in Prisma Client, so this should never be reached"),
```

Comment:
I'd like to match the offical Prisma client's [unsupported handling](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#unsupported), where `create` and `update` aren't generated for models containing required fields with an unsupported type. Making this function return `Option<TokenStream>` would probably be a good start.